ecosystem/node-checker/src/evaluators/direct/latency.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use super::DirectEvaluatorInput;
+use crate::{
+    configuration::EvaluatorArgs,
+    evaluator::{EvaluationResult, Evaluator},
+};
+use anyhow::Result;
+use clap::Parser;
+use poem_openapi::Object as PoemObject;
+use serde::{Deserialize, Serialize};
+use thiserror::Error as ThisError;
+use tokio::time::{Duration, Instant};
+use url::Url;
+
+pub const CATEGORY: &str = "performance";
+
+#[derive(Debug, ThisError)]
+pub enum LatencyEvaluatorError {}
+
+#[derive(Clone, Debug, Deserialize, Parser, PoemObject, Serialize)]
+pub struct LatencyEvaluatorArgs {
+    /// The number of times to hit the node to check latency.
+    #[clap(long, default_value_t = 5)]
+    pub num_samples: u16,
+
+    /// The delay between each call.
+    #[clap(long, default_value_t = 20)]
+    pub delay_between_samples_ms: u64,
+
+    /// The number of responses that are allowed to be errors.
+    #[clap(long, default_value_t = 1)]
+    pub num_allowed_errors: u16,
```

Comment:
nit: do we think we'll need this?

Reply:
Perhaps not, I figured it's just nice to have the option, I think failing them for a single missed response is a bit brutal.